mono/dis/util.c

```diff
@@ -20,9 +20,8 @@
 #endif
 
 /**
- * map:
- * @code: code to lookup in table
- * @table: table to decode code
+ * \param code code to lookup in table
```

Comment:
I don’t know. I thought we weren’t generating anything from the runtime sources. Also, that URL doesn’t resolve; are you referring to http://www.mono-project.com/docs/ or something else?